packages/schematics/update/update/npm.ts

```diff
@@ -44,18 +113,10 @@ function getNpmConfigOption(
     return value;
   }
 
-  const subject = new ReplaySubject<string | undefined>(1);
-
-  const optionValue = npmConfig && npmConfig[fullOption];
-  if (optionValue == undefined || optionValue == null) {
-    subject.next();
-  } else {
-    subject.next(optionValue);
-  }
-
-  subject.complete();
+  value = fullOption == '_authToken'
```

Comment:
this should probably be `option.startsWith('_')`

Reply:
Done. Investigating CI failures then I'll repush.

Reply:
That should fix at least some of them.